[Merged by Bors] - Depricated and Unstable config flags (#683)
metalbear-co/mirrord: mirrord/config/derive/README.md

    - `nested` to use a value of a type that itself implement `MirrordConfig`. The value used is its `Generated` type.
    - `env = &str` to load the value from the specified environment variable, if it's populated.
    - `default = &str` to set a default value to the field. This also implicitly `unwrap`s it.

>     - `unstable` mark field as unstable and print a error message to the client

Reviewer: ```suggestion
    - `unstable` mark field as unstable and print an error message to the user
```
